x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -589,19 +633,47 @@ export const ModelsList: FC<Props> = ({
     },
     {
       width: '10%',
-      field: 'state',
       name: i18n.translate('xpack.ml.trainedModels.modelsList.stateHeader', {
         defaultMessage: 'State',
       }),
       align: 'left',
       truncateText: false,
-      render: (state: ModelState) => {
+      render: ({ state, downloadState }: ModelItem) => {
         const config = getModelStateColor(state);
-        return config ? (
+        if (!config) return null;
+
+        const isDownloadInProgress = state === MODEL_STATE.DOWNLOADING && downloadState;
+
+        const label = (
           <EuiHealth textSize={'xs'} color={config.color}>
             {config.name}
           </EuiHealth>
-        ) : null;
+        );
+
+        return (
+          <EuiFlexGroup direction={'column'} gutterSize={'none'}>
+            {isDownloadInProgress ? (
+              <EuiFlexItem>
+                <EuiProgress
+                  label={label}
+                  valueText={
+                    <>
+                      {((downloadState.downloaded_parts / downloadState.total_parts) * 100).toFixed(
+                        0
+                      ) + '%'}
+                    </>
+                  }
+                  value={downloadState?.downloaded_parts}
+                  max={downloadState?.total_parts}
+                  size="xs"
+                  color={config.color}
```

Comment:
hey @joana-cps! I use the same color as we have in `EuiHealth` for downloading state, but we're concerned it looks a bit like a warning color 🤔 Do you think it's worth trying something else?

Reply:
hey @darnautov, I agree with you, we use yellow for all warning cases. My suggestion, in this case, is to change to a more neutral color like our primary or subdued documented as [EuiProgress](https://eui.elastic.co/#/display/progress#colors).

Reply:
thx @joana-cps, updated to a `primary` color in https://github.com/elastic/kibana/pull/184906/commits/d2b9dadb12eec1b9a73710c2209900506d42f84d